etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/ruleengine/transformation/custom/RemovePatientIdentifiers.java

```diff
@@ -15,7 +16,12 @@ public class RemovePatientIdentifiers implements CustomFhirTransformation {
     @Override
     public void transform(HealthData<?> resource, Map<String, Object> args) {
         Bundle bundle = (Bundle) resource.getUnderlyingData();
-        HapiHelper.setPID3_4Value(bundle, ""); // remove PID.3-4
-        HapiHelper.setPID3_5Value(bundle, ""); // remove PID.3-5
+
+        Identifier patientIdentifier = HapiHelper.getPID3Identifier(bundle);
```

Comment:
Can lines 19 to 24 be extracted to `HapiHelper.setPID3_4Value`?

Reply:
Refactored.